.editorconfig

```diff
@@ -6,7 +6,7 @@ root = true
 
 
 [*]
-end_of_line = lf
+# end_of_line = lf (https://github.com/editorconfig/editorconfig/issues/226)
```

Comment:
I generally prefer PRs to be single purpose. Can you open a separate PR with these dotfile changes?